client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.html

```diff
@@ -162,47 +162,47 @@ <h1 class="sr-only" i18n>Configuration</h1>
             <div class="form-group form-group-right col-12 col-lg-8 col-xl-9">
 
               <div class="form-group">
-                <label i18n for="instanceAdministrator">Who is behind the instance?</label>
+                <label i18n for="instanceAdministrator">Who is behind the instance?</label><my-help helpType="markdownText"></my-help>
                 <div i18n class="label-small-info">A single person? A non-profit? A company?</div>
 
                 <my-markdown-textarea
-                  name="instanceAdministrator" formControlName="administrator" textareaMaxWidth="500px" textareaHeight="75px"
```

Comment:
I think removing `textareaHeight="75px"` will adapt the textarea height to the text height. If you have a very long text, your text might hard to scroll with the rest of the page on laptop and small screens.

Reply:
Seen below: the default value is 150px.